mxd/modules/connector/values.yaml

```diff
@@ -24,6 +24,10 @@
 participant:
   id: "test-participant"
 controlplane:
+#  livenessProbe:
```

Comment:
any reason you added this?

Reply:
When I do remote debug and put a breakpoint for a long time in our connector's data-plane / control-plane, kubernetes probes fail and it restarts the pod.
So I often need these configs to disable the probes. Just that, I had to copy and paste every time, now I have to just uncomment.

Reply:
we should not have commented code in there. i would prefer to have it active, and you can comment it out when debugging. pls uncomment.

Reply:
I enabled it to `true` (which is also the default value in helm chart) and while debugging, I can make it `false`.